staff/views/repos.py

```diff
@@ -147,113 +259,27 @@ def get_form_kwargs(self):
 
 
 @method_decorator(require_role(CoreDeveloper), name="dispatch")
-class RepoList(View):
-    get_github_api = staticmethod(_get_github_api)
-
-    @csp_exempt
```

Comment:
Do we need to make the new view `csp_exempt`?

Reply:
Nope, that's so the sort&filter JS on the dashboard can be loaded.